library-app/src/App.tsx

```diff
@@ -2,11 +2,23 @@ import Homepage from './components/pages/Homepage/Homepage'
 import styles from './App.module.css'
 import { Route, Routes } from 'react-router-dom'
 import Login from './components/pages/Login/Login'
+import React, { useEffect, useState } from 'react'
+import PrivateRoutes from './router/PrivateRoutes'
+import BookDetails from './components/pages/BookDetails'
 
 function App() {
+  const [accessToken, setAccessToken] = useState(localStorage.getItem('accessToken'))
```

Comment:
You can create separate file to work with token instead of directly working with localStorage each time you want to get / set token. Also, 'token' key shouldn't be hardcoded like this - you can have some constants with keys

Reply:
okay, done.